aptos-move/aptos-release-builder/src/main.rs

```diff
@@ -27,11 +29,29 @@ pub enum Commands {
         #[clap(short, long)]
         release_config: PathBuf,
         #[clap(short, long)]
-        test_dir: PathBuf,
-        #[clap(short, long)]
         endpoint: url::Url,
         #[clap(long)]
         framework_git_rev: Option<String>,
+        #[clap(subcommand)]
+        input_option: InputOptions,
+        #[clap(long)]
+        mint_to_validator: bool,
+    },
+}
+
+#[derive(Subcommand, Debug)]
+pub enum InputOptions {
+    FromDirectory {
+        #[clap(short, long)]
+        test_dir: PathBuf,
+    },
+    FromArgs {
+        #[clap(long)]
```

Comment:
Can you maybe add quick comment / doc for these option?

Reply:
Comments added!